google/cloud/firestore_v1/query.py

```diff
@@ -333,16 +339,17 @@ def order_by(self, field_path, direction=ASCENDING):
             field_filters=self._field_filters,
             orders=new_orders,
             limit=self._limit,
+            limit_to_last=self._limit_to_last,
             offset=self._offset,
             start_at=self._start_at,
             end_at=self._end_at,
             all_descendants=self._all_descendants,
         )
 
     def limit(self, count):
-        """Limit a query to return a fixed number of results.
+        """Limit a query to return first `count` results.
```

Comment:
I think this is missing an article. "the first `count` results". 

Also, is it the first, or that many results from the point of the query? For instance, if the query is set to skip (offset) the first 100 results, this might be misleading

Reply:
Hm-m, looks like I've missed the word "matching":

> Limit a query to return the last `count` matching results.

Does this make more sense?

Reply:
It does, though still wondering if last and first make sense? it could actually be the middle `count` results.

For instance, if I offset by 10, limit to 10, but have 30 results, I actually am viewing results 10-20

Reply:
Perhaps `Limit a query to return the next `count` matching results.`?